packages/xrpl/test/integration/transactions/checkCreate.test.ts

```diff
@@ -0,0 +1,53 @@
+import { assert } from 'chai'
+import { CheckCreate } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import { generateFundedWallet, testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('CheckCreate', () => {
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'base',
+    async () => {
+      const wallet2 = await generateFundedWallet(testContext.client)
+      const tx: CheckCreate = {
+        TransactionType: 'CheckCreate',
+        Account: testContext.wallet.classicAddress,
+        Destination: wallet2.classicAddress,
+        SendMax: '50',
+      }
+
+      await testTransaction(testContext.client, tx, testContext.wallet, {
+        count: 1,
+        delayMs: 1000,
+      })
```

Comment:
nit: Is the retry count 0-indexed maybe it'd be better to call it `retryCount` in that case so it matches the number of retries left.